services/movie_session.py

```diff
@@ -0,0 +1,42 @@
+from datetime import datetime
+
+from db.models import MovieSession, CinemaHall
+from services.movie import get_movie_by_id
+
+
+def create_movie_session(movie_show_time: datetime,
+                         movie_id: int,
+                         cinema_hall_id: int) -> None:
+    MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id
+    )
+
+
+def get_movies_sessions(session_date: datetime = None) -> list[MovieSession]:

```

Comment:
It returns  QuerySet
```suggestion
def get_movies_sessions(session_date: datetime = None) ->  QuerySet[MovieSession]:

```